basics/getting_started/deploying_plugin.md

```diff
@@ -15,7 +15,8 @@ To deploy a plugin:
 
   ![Jar Saved Notification](deploying_plugin/img/jar_saved_notification.png)
 
-* Copy the newly created archive file to the `.IntelliJIDEAx0\config\plugins` folder, and then restart your IDE so the changes may take effect.  To know how to locate your *plugins* directory, refer to [IDE Settings, Caches, Logs, and Plugins](/basics/settings_caches_logs.md).
+* Copy the newly created archive file to the `.IntelliJIDEAx0\config\plugins` folder, and then restart your IDE so the changes may take effect.  To know how to locate your *plugins* directory,
```

Comment:
I think it's better to tell people to use "Install plugin from disk", so that they won't need to locate the directory manually. Also, newly installed plugins are enabled by default, so there is no need to tell people to enable the plugin as a separate step.